softaware.Authentication.Hmac.AspNetCore/HmacAuthenticationHandler.cs

```diff
@@ -106,7 +106,7 @@ private async Task<bool> IsValidRequestAsync(HttpRequest req, byte[] body, strin
         {
             var requestContentBase64String = string.Empty;
             var absoluteUri = string.Concat(
-                        req.Scheme,
+                        GetRequestScheme(req),
```

Comment:
Please conform to our coding guidlines and add the `this` keyword. Thank you.

```
this.GetRequestScheme(req)
```

Reply:
done